db/models.py

```diff
@@ -50,3 +55,54 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE,
+                             related_name="orders")
```

Comment:
The `Order` model uses a `ForeignKey` to the `User` model. Ensure that the `User` model is correctly set as the custom user model in your Django settings using `AUTH_USER_MODEL`. This is crucial for maintaining the integrity of user-related data.